### PR TITLE
Troubleshooting tip Keeping ZSS secure with TLS #1883

### PR DESCRIPTION
**Pull request by AbhishekJamhoriya**
Signed-off-by: AbhishekJamhoriya jamhoriya.1@iitj.ac.in

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/user-guide/mvd-configuration.md [hunk 1]
```diff
@@ -452,6 +452,24 @@ The following steps assume you have installed a Zowe runtime instance (which inc
...
+ ** WARNING: Connection is insecure! TLS needed but not found on socket. **
+```
+
+ZSS is a webserver that performs certain system calls for data management and even security. It is important to have a configuration that keeps the network traffic for ZSS secure at all times. When TLS is not active on ZSS, at startup and periodically the warning will be shown. If there is no need for ZSS communication to a client outside the z/OS system, it is possible to make ZSS accessible only internal to z/OS, but if TLS is not activated, the warning will remain.
+
+There are two ways to ensure ZSS has TLS.
+The default, when the environment variable ZOWE_ZSS_SERVER_TLS=true is set. This will use the Zowe keystore for TLS certificate management.
+
+AT-TLS, as described here: https://docs.zowe.org/stable/user-guide/mvd-configuration/#configuring-zss-for-https
+Note that AT-TLS and ZOWE_ZSS_SERVER_TLS=true are mutually exclusive, but perform the same task of making ZSS use HTTPS.
```
Comment (timgerstel) [inline docs/user-guide/mvd-configuration.md:469]: I would rephrase this to say, "AT-TLS must be disabled if TLS is enabled, and vice versa" instead of "mutually exclusive" just for clarity.

**Comment by balhar-jakub** [comment]
Thanks for your contribution, we should hae integrated it sooner, but as of now, we reworked the whole TLS and AT-TLS related documentation and this isn't really valid anymore.
